docs/developer-guide/nodejs-api.md

```diff
@@ -147,6 +147,38 @@ console.log(code.text);     // "var foo = bar;"
 
 In this way, you can retrieve the text and AST used for the last run of `linter.verify()`.
 
+### verifyAndFix()
+
+This method is similar to `verify` method when it comes to verifying the code against the rules but it also runs the fixer logic from all the fix compatible rules and return the object which has the fixed source code in it.
+The lifecycle is that it verifies the code and then runs the fix on it and this is done at most 10 times right now.
```

Comment:
I don't think it's necessary to mention the number of passes here, because it's an implementation detail and could change without warning in the future. 

We might also want to explain what messages appear (for example, problems that were fixed will not be reported).

Maybe this paragraph could be reworded:

> This method is similar to `verify` except that it also runs autofixing logic, similar to the `--fix` flag on the command line. The result object will contain the autofixed code, along with any remaining linting messages for the code that were not autofixed.